tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -189,18 +189,18 @@ To resume the stream, click the **Follow Latest** button that appears after paus
 
 ### <a id="filter-by-container"></a>Filtering by container
 
-To display logs for a specific container only, select the desired container from the **Container**
-drop-down menu. If you do not select a container, logs for all containers within the pod are displayed.
+To display logs for a different container, select the desired container from the **Container**
+drop-down menu.
 
 ### <a id="filter-by-date-and-time"></a>Filtering by date and time
 
-To see all logs since a specific date and time, select or type the UTC timestamp in the
+To see logs since a specific date and time, select or type the UTC timestamp in the
 **Since date** field.
 If no logs are displayed, adjust the timestamp to an earlier time.
 If you do not select a timestamp, all logs produced since the pod was created are displayed.
 
 For optimal performance, the pod logs page limits the total log entries displayed to the most recent
-10,000.
+10,000 (approximately).
```

Comment:
The reason for the "approximately" is that the 10k limit is for any log entries as well as inline notifications that appear (these notifications are generated by the UI and are not technically log "entries").

Reply:
i.e. `total_notifications + total_entries < 10,000`

Reply:
I don't think it's worth it to confuse customers with the details I mentioned, but perhaps just use the verbiage "approximate" to indicate that it may not always be exact. I welcome any thoughts/feedback though :)

Reply:
OK, I added a new commit with an alternative. Thoughts welcome.